src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -1314,7 +1314,7 @@ public void LanguageVersion_GetErrorCode()
         {
             var versions = Enum.GetValues(typeof(LanguageVersion))
                 .Cast<LanguageVersion>()
-                .Except(new[] { LanguageVersion.Default, LanguageVersion.Latest })
+                .Except(new[] { LanguageVersion.Default, LanguageVersion.Latest, LanguageVersion.CSharp7_2 })
```

Comment:
:memo: I didn't add a generic error code for the new language version. I think we should use specific errors instead.